branch_cycles/task2.js

```diff
@@ -0,0 +1,126 @@
+const averageGrade = 85;
+switch (averageGrade) {
+    case 60:
+        console.log("Задовільно");
+        break;
+    case 61:
+        console.log("Задовільно");
+        break;
+    case 62:
+        console.log("Задовільно");
+        break;
+    case 63:
+        console.log("Задовільно");
+        break;
+    case 64:
+        console.log("Задовільно");
+        break;
+    case 65:
+        console.log("Задовільно");
+        break;
+    case 67:
+        console.log("Задовільно");
+        break;
+    case 68:
+        console.log("Задовільно");
+        break;
+    case 69:
+        console.log("Задовільно");
+        break;
+    case 70:
+        console.log("Задовільно");
+        break;
+    case 71:
+        console.log("Добре");
+        break;
+    case 72:
+        console.log("Добре");
+        break;
+    case 73:
+        console.log("Добре");
+        break;
+    case 74:
+        console.log("Добре");
+        break;
+    case 75:
+        console.log("Добре");
+        break;
+    case 76:
+        console.log("Добре");
+        break;
+    case 77:
+        console.log("Добре");
+        break;
+    case 78:
+        console.log("Добре");
+        break;
+    case 79:
+        console.log("Добре");
+        break;
+    case 80:
+        console.log("Добре");
+        break;
+    case 81:
+        console.log("Дуже Добре");
+        break;
+    case 82:
+        console.log("Дуже Добре");
+        break;
+    case 83:
+        console.log("Дуже Добре");
+        break;
+    case 84:
+        console.log("Дуже Добре");
+        break;
+    case 85:
+        console.log("Дуже Добре");
+        break;
+    case 86:
+        console.log("Дуже Добре");
+        break;
+    case 87:
+        console.log("Дуже Добре");
+        break;
+    case 88:
+        console.log("Дуже Добре");
+        break;
+    case 89:
+        console.log("Дуже Добре");
+        break;
+    case 90:
+        console.log("Дуже Добре");
+        break;
+    case 91:
+        console.log("Відмінно");
+        break;
+    case 92:
+        console.log("Відмінно");
+        break;
+    case 93:
+        console.log("Відмінно");
+        break;
+    case 94:
+        console.log("Відмінно");
+        break;
+    case 95:
+        console.log("Відмінно");
+        break;
+    case 96:
+        console.log("Відмінно");
+        break;
+    case 97:
+        console.log("Відмінно");
+        break;
+    case 98:
+        console.log("Відмінно");
+        break;
+    case 99:
+        console.log("Відмінно");
+        break;
+    case 100:
+        console.log("Відмінно");
+        break;
+
+    default:
+        console.log("Незадовільно");
+}
```

Comment:
:)

Reply:
use ranges for comparing, what you would do if average grade is float point number?